sycl/unittests/kernel-and-program/MultipleDevsCache.cpp

```diff
@@ -210,5 +212,7 @@ TEST_F(MultipleDeviceCacheTest, ProgramRetain) {
     EXPECT_EQ(KernelCache.size(), (size_t)2) << "Expect 2 kernels in cache";
   }
   // Cache is cleared here, check kernel release
+  // 3 kernel releases is expected because kernel_bundle::get_kernel() calls piKernelRetain
+  // so one more kernel release is needed
   EXPECT_EQ(KernelReleaseCounter, 3) << "Expect 3 piKernelRelease calls";
```

Comment:
Is it logically equal to `KernelCount * DeviceCount +  1`? I believe, this will be more descriptive as current comment doesn't answer why exactly 3 releases are performed.

Reply:
I added additional comments about count of kernel releases